src/utils/utils.js

```diff
@@ -48,3 +50,18 @@ export function segmentizeMatch(text, search) {
         }
     })
 }
+
+/**
+ * Parse an RGB color
+ *
+ * @param {string} color Color code in string format (should be between 0 and 255)
+ * @returns {Number} Color code, default to 255 in case of invalid
+ */
+export function parseRGBColor(color) {
+    try {
+        return Math.max(Math.min(Number(color), 255), 0)
+    } catch (error) {
+        log.error(`Invalid RGB color code ${color}`)
```

Comment:
you could log the error here (it's unused currently)
`log.error('Invalid RGB color code', color, error)`